### PR TITLE
fix cita_monitor_agent.py for hex_number initialization issue

### PR DESCRIPTION
**Pull request by flyq**
for detail: https://talk.citahub.com/t/topic/1228/3

### CODE REVIEW

File: agent/cita_monitor_agent.py [hunk 1]
```diff
@@ -303,6 +303,8 @@ def exporter():
     else:
         print(genesis_block_info)
     block_number_info = class_result.block_number()
+    hex_number = 0x1
+    previous_hex_number = 0x1
```
Comment (rainchen) [inline agent/cita_monitor_agent.py]: `previous_hex_number` means `hex_number-1`, so here it should be `0x0`.

**Review by rainchen — CHANGES_REQUESTED** [review]
thx for you fix, and please update your codes according to the review comments.

File: agent/cita_monitor_agent.py (hunk 1 above)
Comment (rainchen) [reply agent/cita_monitor_agent.py]: @flyq can you update this？

Comment (flyq) [reply agent/cita_monitor_agent.py]: already update it

Comment (rainchen) [reply agent/cita_monitor_agent.py]: please use amend to squash your commits, see https://help.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message

Comment (rainchen) [reply agent/cita_monitor_agent.py]: can you amend your commits?

Comment (flyq) [reply agent/cita_monitor_agent.py]: I don't know how to amend the commits, so I think it is a better way that closing this pr and you push a commit to fix this. @rainchen

**Comment by rainchen** [comment]
@flyq why did you closing this issue?

File: agent/cita_monitor_agent.py (hunk 1 above)
Comment (rainchen) [reply agent/cita_monitor_agent.py]: @flyq try these steps:

1. git checkout -b "fix-agent-hex-humber-init-issue"
2. git reset --soft HEAD^
3. git reset --soft HEAD^
4. git add agent/cita_monitor_agent.py
5. git commit -am "fix cita_monitor_agent.py for hex_number initialization issue"
6. git push origin fix-agent-hex-humber-init-issue

Comment (rainchen) [reply agent/cita_monitor_agent.py]: @flyq would you please handle this? we're going to release a new version, will be great to have this issue fixed

**Review by rainchen — APPROVED** [review]

File: agent/cita_monitor_agent.py (hunk 1 above)
Comment (flyq) [reply agent/cita_monitor_agent.py]: @rainchen thx for your steps, I follow your advice: https://github.com/citahub/cita-monitor/pull/162

PS: I will learn more about git

**Pull request merged**